docs/admin/_kubeadm/kubeadm_alpha_phase_addon_kube-dns.md

```diff
@@ -0,0 +1,22 @@
+
+Install the kube-dns addon to a Kubernetes cluster.
+
+### Synopsis
+
+
+Install the kube-dns addon to a Kubernetes cluster.
+
+```
+kubeadm alpha phase addon kube-dns
+```
+
+### Options
+
+```
```

Comment:
same - especially seeing how alpha cmd line features change.

Reply:
same :-) 
by the way, I'm going to add alpha disclaimer where missing in kubeadm code

Reply:
Yes, adding the alpha disclaimer would be great!